docs/customizing-lampray.md

```diff
@@ -0,0 +1,134 @@
+# Customizing Lampray
+
+> Learn how to customize Lampray's fonts, transparency, color theme, and more.
+
+## Prerequisites
```

Comment:
Update to: "Install Lampray or build from source" and point to the relevant docs.